apps/website/src/pages/api/short-links/track-click.ts

```diff
@@ -1,20 +1,27 @@
 import type { NextApiRequest, NextApiResponse } from "next";
 import { z } from "zod";
+import { track } from "@vercel/analytics/server";
+
 import { prisma } from "@/server/db/client";
 
-const trackClickSchema = z.object({ id: z.string().cuid() });
+const trackClickSchema = z.object({
+  id: z.string().cuid(),
+  slug: z.string(),
+  link: z.string(),
+});
 export type TrackClickSchema = z.infer<typeof trackClickSchema>;
 
 //API for short links tracking
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse,
 ) {
-  const id = req.body.id;
+  const { id, slug, link } = req.body;
   await prisma.shortLinksTracking.upsert({
     where: { id: id },
     create: { id, clicks: 1 },
     update: { clicks: { increment: 1 } },
   });
+  await track("Short link clicked", { id, slug, link });
```

Comment:
Looking at https://vercel.com/docs/analytics/limits-and-pricing#limits-per-plan, it looks like we can't even track id + slug + link here, as we're limited to two properties